lib/fetch/sanitize.spec.js

```diff
@@ -288,8 +288,115 @@ describe('data/sanitize', () => {
         github: 'http://github.com',
         twitter: 'http://twitter.com',
         categories: [
-          'Engineering',
+          { name: 'Engineering', slug: 'engineering' },
+          { name: 'Leadership', slug: 'leadership' },
+        ],
+      });
+    });
+
+    it('converts valid tags into categories', () => {
```

Comment:
Oh right I remember now - we wanted multiple assignable categories and tags are the only way 👍